mautic/mautic
# Hide Is Unique Identifier for company fields as far as this is not supported

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | #7316




#### Description:
Companies doesn't use Is Unique Identifier for fields in any places. 
Add this feature it's not easy, then it's better hide this option for this time and avoid confused users.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try create company field with Is Unique Identifier - this options should be hidden now
3. This PR also set flag to 0 for all companies Is Unique Identifier fields  with migrations
